docs/source/quick_start.rst

```diff
@@ -42,14 +43,23 @@ with the column names that we want to specify as categorical. For this example,
       "Married since": pl.DateTime,
    }
 
-To finish loading the dataset, we simply use the :meth:`polars.read_csv` function, passing in our CSV file and the ``data_types`` dictionary as an argument. 
+To finish loading the dataset, we can either use the :meth:`polars.read_csv` function or the ``metasyn``
+:func:`metasyn.filereader.read_csv` function. The advantage of the latter is that with it metasyn will remember the
+specifics of the file format of your file (for example if your separator between columns was a `,` or `;`)
+and when writing the synthetic data file, this can be reused.
+
+.. note::
+   Metasyn also supports sav files (:func:`metasyn.filereader.read_sav`) and tsv files (:func:`metasyn.filereader.read_tsv`).
+
+For reading a CSV file, you should give the ``data_types`` dictionary as an argument. 
 
 .. code-block:: python
 
-   df = pl.read_csv(csv_path, schema_overrides=data_types)
+   df, file_reader = ms.read_csv(csv_path, schema_overrides=data_types)
 
 
-This converts the CSV file into a DataFrame named ``df``.
+This converts the CSV file into a DataFrame named ``df``. Additionally, we have a ``file_reader`` object
```

Comment:
this will change to file_format dictionary

Reply:
Done